sql/derived_tables/loans_items.sql

```diff
@@ -29,7 +29,7 @@ SELECT
     cl.loan_date,
     cl.due_date AS loan_due_date,
     cl.return_date AS loan_return_date,
-    json_extract_path_text(cl.data, 'systemReturnDate') AS system_return_date,
+    json_extract_path_text(cl.data, 'systemReturnDate')::timestamptz at time zone 'UTC' AS system_return_date,
```

Comment:
The addition of `at time zone 'UTC'` converts the `timestamptz` to `timestamp without time zone`.  It would probably be better to leave it as `timestamptz` in a derived table.

Reply:
Is it worth adding towards the top of the script in the notes something along the lines "By default times are being converted to UTC."?

Reply:
@lib-eric  That seems OK to me, but I would add `_utc` to the end of the column name.  On the other hand, if it is kept as `timezonetz`, users can set their local time zone and have it converted automatically.

Reply:
I think `timestamptz` is really the behavior we intended. We intended to match the conversion that LDP does automatically when it detects a column is a timestamp column, and the inclusion of UTC actually seems to give an incorrect result. Thanks for catching that!